src/lib/adzerk_fetcher.ts

```diff
@@ -0,0 +1,57 @@
+import {AdzerkRequest, AdzerkResponse} from './adzerk';
+
+const API_ENDPOINT = 'https://engine.adzerk.net/api/v2';
```

Comment:
Since we learned in the release notes that this can be given a CNAME, should this be configurable?